typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,19 @@
+let lightSwitch = false;
+
+const $bulb = document.querySelector('.bulb');
+const $bulbBackground = document.querySelector('.bulb-background');
+if (!$bulb) throw new Error('$bulb query has failed');
+$bulb.addEventListener('click', clickHandler);
+function clickHandler(): void {
+  if (!$bulbBackground) throw new Error('$bulbBackground query has failed');
+  if (!$bulb) throw new Error('$bulb query has failed');
+  if (lightSwitch === false) {
+    lightSwitch = true;
+    $bulb.className = 'bulb bulb-on';
+    $bulbBackground.className = 'bulb-background background-on';
```

Comment:
MINOR optimization, you could just check if _not_ `lightSwitch` which will evaluate to a boolean checking if `lightSwitch` is not _truthy_ which evaluates to _falsy_ (which is essentially coerced into `false`)

`if (!lightSwitch)`